client/my-sites/people/edit-team-member-form/index.jsx

```diff
@@ -355,4 +383,18 @@ module.exports = protectForm( React.createClass( {
 			</Main>
 		);
 	}
-} ) );
+}
+
+export default connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+		const site = getSelectedSite( state );
+
+		return {
+			siteId,
+			siteSlug: getSiteSlug( state, siteId ),
```

Comment:
They're pretty equivalent. `getSelectedSiteSlug` is just a combination of `getSelectedSiteId` + `getSiteSlug`, so we're saving a tiny amount of work. I can go either way on this one.